packages/api/helper/mailer.js

```diff
@@ -54,23 +54,23 @@ function sendDeclineEmail(nomination) {
 }
 
 function sendSurveyEmail(nomination) {
+
   email
     .send({
-      template: 'survey',
-      attachments: './survey/header.jpg',
-      message: {
-        from: adminEmail,
-        to: nomination.representativeEmailAddress,
-      },
-      locals: {
-        name: nomination.representativeName,
-        patientName: nomination.patientName,
-        email: nomination.representativeEmailAddress,
-        imgUrl,
-      },
-    })
-    .catch((err) => console.log(err))
-    .then(() => console.log('email has been sent!'));
+    template: 'survey',
+    attachments: './survey/header.jpg',
+    message: {
+      from: adminEmail,
+      to: nomination.representativeEmailAddress
+    },
+    locals: {
+      name: nomination.representativeName,
+      patientName: nomination.patientName,
+      email: nomination.representativeEmailAddress,
+      imgUrl
+    }
+  }).catch((err) => console.log(err))
+  .then(() => console.log('email has been sent!'));
```

Comment:
I'm sure this is prettier or some other thing "fixing" the formatting here, not something you did. But it's actually reduced the readability and introduced some style inconsistencies.

1) Having the catch on a separate line aids in readability, especially with such a long object passed to the method in the first place
2) The indentation now is a little wonky; we could fix this by moving the `.send` call up a line, so it becomes `email.send({`
3) We generally use [trailing commas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) in literals, like on line 64, 70, and 71 here. This is a common practice in a lot of languages to avoid a certain class of bug.

Generally speaking, it looks like a lot of this PR is prettier (or some other formatter) making changes. Some of them are good, but some of them amount to personal preference. The next person to edit these files with their own personal settings will produce different changes, which will create inconsistency. I think the solution is either to not use prettier or to establish some shared configuration, but that may be a problem for us to solve as a group...

Reply:
I believe that this is fixed in the new commit